packages/@aws-cdk/aws-securityhub/package.json

```diff
@@ -62,17 +62,6 @@
     "url": "https://aws.amazon.com",
     "organization": true
   },
-  "jest": {
```

Comment:
Why are you removing this? I would prefer 'jest' tests to nodeunit tests, if we are starting fresh anyway.

Reply:
I have enabled it again.